styles.css

```diff
@@ -329,3 +340,178 @@ footer {
   opacity: 1;
   right: 0;
 }
+
+.nobr {
+  display: none;
+}
+
+.desk-icons {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .skills {
+    flex-wrap: nowrap;
+    justify-content: space-around;
+    width: 447px;
+    margin: auto;
+    background-image: none;
+  }
+
+  .skills2 {
+    text-align: center;
+    padding: 0;
+    flex: 20%;
+  }
+
+  .bio {
+    width: 447px;
+    margin: auto;
+  }
+
+  .divider {
+    width: 448px;
+  }
+
+  .divider2 {
+    width: 90%;
+  }
+
+  .br {
+    margin-left: 102px;
+    display: none;
+  }
+
+  .nobr {
+    display: block;
+    font-style: normal;
+    font-size: 64px;
+    font-weight: 700;
+    line-height: 114px;
+    color: #9390ff;
+    padding-left: 2%;
+  }
+
+  .post {
+    display: grid;
+    justify-items: center;
+    grid-template-columns: 2fr 1fr;
+    gap: 24px;
+    padding-top: 88px;
+    max-width: 1100px;
+  }
+
+  .multipost {
+    display: flex;
+    align-items: center;
+    padding-left: 142px;
+    padding-right: 142px;
+  }
+
+  .img-post {
+    max-width: 684px;
+    max-height: 448px;
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+    justify-self: end;
+    margin: 0;
+    background-image: url(./images/Snapshoot_Portfolio@2x.png);
+    background-repeat: no-repeat;
+    resize: both;
+    overflow: auto;
+  }
+
+  .contact {
+    margin-top: 228px;
+    max-width: 684px;
+  }
+
+  #finalbutton {
+    margin: auto;
+    margin-top: 101px;
+    width: 250px;
+    height: 40px;
+  }
```

Comment:
[OPTIONAL] You did not address the previous reviewer's requirements, please make sure you implement the required button styling.